cli.py

```diff
@@ -9,7 +10,9 @@
 def cli(lang):
     ob = GIG()
     if lang == 'None':
-        click.echo('Generating empty gitignore'
-        'since `lang` argument not provided')
+        click.echo(click.style("[INFO] Empty GITIGNORE generated,"
+                               "if one doesn\'t already exists.", fg='blue'))
```

Comment:
change it to
`since lang argument not provided`

Reply:
So you want to remove the feature of adding a blank ```.gitignore``` for no ```--lang``` arg provided ?